ui/DesignSystem/Primitive/Button.svelte

```diff
@@ -320,9 +320,7 @@
   on:click
   on:mouseenter
   on:mouseleave
-  {style}>
+  style={`${style}` + "gap: .5rem;"}>
```

Comment:
Shouldn't the `gap` property be added to the `button` class?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1928/commits/6e8a1b09c376d1fb368842365054e938de5aefe7